omnibus-arbitrum-mainnet.toml

```diff
@@ -204,3 +204,69 @@ args = [
     18,
     "<%= imports.system.contracts.USDRouter.address %>",
```

Comment:
confirming this is not supposed ot be "USDC"

Reply:
Yeah, this is a contract of the internal stable (snxUSD)